app/uk/gov/hmrc/partnershipidentificationfrontend/models/RegistrationStatus.scala

```diff
@@ -56,7 +65,10 @@ object RegistrationStatus {
             businessPartnerId => Registered(businessPartnerId)
```

Comment:
The other case sections are returning JsSuccess(Some registration status, path). Should the success section do this also? There may be an implicit conversion.